module/ldbc-query-builder/src/main/scala/ldbc/query/builder/Table.scala

```diff
@@ -12,12 +12,14 @@ import scala.compiletime.*
 import scala.compiletime.ops.int.*
 import scala.annotation.targetName
 
+import ldbc.sql.ResultSet
 import ldbc.dsl.*
+import ldbc.dsl.codec.{ Encoder, Decoder }
 import ldbc.query.builder.statement.*
 import ldbc.query.builder.interpreter.*
 import ldbc.query.builder.formatter.Naming
 
-sealed trait MySQLTable[P <: Product]:
+trait MySQLTable[P <: Product]:
```

Comment:
![must](https://img.shields.io/badge/review-must-critical.svg) 
Return to seald